libs/core/src/lib/utils/services/content-density.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { BehaviorSubject } from 'rxjs';
+
+export type ContentDensity = 'cozy' | 'condensed' | 'compact';
+
+@Injectable()
+/**
+ * Service taking care of ContentDensity
+ */
+export class ContentDensityService {
+    contentDensity: BehaviorSubject<ContentDensity>;
```

Comment:
it seems it's a single reason service, and you could hide `contentDensity`
```suggestion
    private readonly contentDensity: BehaviorSubject<ContentDensity> =  new BehaviorSubject('cozy');;
```